src/robot.coffee

```diff
@@ -211,6 +211,13 @@ class Robot
   loadFile: (path, file) ->
     ext  = Path.extname file
     full = Path.join path, Path.basename(file, ext)
+    if  require.cache[require.resolve(full)]
+      try
+        cacheobj = require.resolve(full)
+        @logger.debug "require cache for #{cacheobj} invalidated."
+        delete require.cache[cacheobj]
+      catch error
+        @logger.error "Unable to invalidate #{cacheobj}: #{error.stack}"
```

Comment:
I personally haven't, but i considered that it would be possible for whatever reason which is the reason for the try/catch. @technicalpickles  i'd be fine with moving this below the require.extensions[ext], but since there's no require cache generated at initial load the if won't ever be true during the initial load. 

So i think this'd mainly be a cosmetic change instead of a functional one.